aws/sdk/examples/applicationautoscaling/src/bin/describe-scaling-policies.rs

```diff
@@ -51,18 +68,7 @@ async fn main() -> Result<(), Error> {
 
     let client = Client::new(&shared_config);
 
-    let response = client
-        .describe_scaling_policies()
-        .service_namespace(ServiceNamespace::Ec2)
-        .send()
-        .await?;
-    if let Some(policies) = response.scaling_policies {
-        println!("Auto Scaling Policies:");
-        for policy in policies {
-            println!("{:?}\n", policy);
-        }
-    }
-    println!("Next token: {:?}", response.next_token);
+    show_policies(&client).await.unwrap();
 
     Ok(())
```

Comment:
Do we want to bubble errors up, or unwrap on them?
```suggestion
    show_policies(&client).await
```